src/draftTransaction.ts

```diff
@@ -1,6 +1,6 @@
 import { BigNumber } from "bignumber.js";
 
-export class TransactionIntent {
+export class DraftTransaction {
```

Comment:
Regarding this renaming, don't you think would be better to rename also all the variables containing the "intent" word? 
Eg. 

`const scIntentFactory = new SmartContractTransactionsFactory({...});` => `const scDraftTransactionFactory = new SmartContractTransactionsFactory({...});`
        
`const intent = scIntentFactory.createTransactionForDeploy({...}) `=> `const draftTransaction = scIntentFactory.createTransactionForDeploy({...}) `

cc @andreibancioiu

Reply:
Indeed. Will rename them.